telescope.md

```diff
@@ -9,6 +9,8 @@
 - [Filtering](#filtering)
     - [Entries](#filtering-entries)
     - [Batches](#filtering-batches)
+- [Tagging](#tagging)
+    - [Adding custom tags](#tagging-adding)
```

Comment:
Please name the anchor the same as the title.

```suggestion
    - [Adding custom tags](#adding-custom-tags)
```

Reply:
Will keep this in mind next time!